src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -8,14 +8,15 @@
 )
 
 from sentry.models import OnboardingTaskStatus, OrganizationOnboardingTask
+from sentry.receivers import check_for_onboarding_complete
 
 
 class OrganizationOnboardingTaskEndpoint(OrganizationEndpoint):
     permission_classes = [OrganizationPermission, ]
 
     def post(self, request, organization):
-        if request.DATA['status'] == 'skipped':
-            OrganizationOnboardingTask.objects.create_or_update(
+        if request.DATA['status'] == 'skipped' and int(request.DATA['task']) in [9, 10]:
```

Comment:
Use a tuple here. `(9, 10)` instead of `[9, 10]`.

Also noting: this `int()` call will possibly fail if bad data were POSTed, for example, if `task` were `a`. it might be safer to just do: `request.DATA['task'] in ('9', '10')` and keep them as strings.